funnel/views/profile.py

```diff
@@ -6,10 +6,11 @@
 from baseframe.forms import render_message, render_redirect, render_form
 from ..models import db, Profile, Team
 from ..forms import NewProfileForm, EditProfileForm
-from .. import app, lastuser
+from .. import funnelapp, app, lastuser
 
 
 @app.route('/new', methods=['GET', 'POST'])
+@funnelapp.route('/new', methods=['GET', 'POST'])
```

Comment:
Remove `@funnelapp.route`. We're removing support for adding new profiles in Talkfunnel.